infra-as-code/bicep/modules/logging/logging.parameters.example.json

```diff
@@ -36,6 +39,16 @@
           "Environment": "POC"
       }
     },
+    "parAutomationAccountTags": {
```

Comment:
Let's emit/remove the `parAutomationAccountTags` and `parLogAnalyticsWorkspaceTags` and leave them to default

Reply:
Remove them totally from parameters file or to have `value: {}` on them?

Reply:
remove them totally so they fall back to defaults in the module which is `parTags` 👍

Reply:
Done